scripts/verify-claimed-stake-fees.ts

```diff
@@ -0,0 +1,206 @@
+import {ethers} from 'hardhat';
+import hre from 'hardhat';
+import {EthersProviderWrapper} from '../deploy/ethers-provider-wrapper';
+import {DelegatorPool, L2Migrator} from '../typechain';
+import {BigNumber} from 'ethers';
+
+const bondingManagerABI = [
+  'function pendingStake(address _delegator, uint256 _endRound) public view returns (uint256)',
+  'function pendingFees(address _delegator, uint256 _endRound) public view returns (uint256)',
+];
+
+const getL1Pending = async (orchAddr: string) => {
+  const bondingManagerAddr = '0x511bc4556d823ae99630ae8de28b9b80df90ea2e';
+  const roundNum = 2466;
+
+  const bondingManager = new ethers.Contract(
+      bondingManagerAddr,
+      bondingManagerABI,
+      new EthersProviderWrapper(hre.companionNetworks['l1'].provider),
+  );
+
+  const [pendingStake, pendingFees] = await Promise.all([
+    await bondingManager.pendingStake(orchAddr, roundNum),
+    await bondingManager.pendingFees(orchAddr, roundNum),
+  ]);
+
+  return {
+    address: orchAddr,
+    stake: pendingStake,
+    fees: pendingFees,
+  };
+};
+
+const getL2Pending = async (orchAddr: string, claimBlock: number) => {
```

Comment:
[optional]: i think that ```getL1Pending``` and ```getL2Pending``` are similar enough to be merged in a single function for reusability's sake - might be useful in other scripts as well.

It could be something like the following:
```
enum ENetwork {
  L1_MAINNET = 'L1_MAINNET',
  L2_MAINNET = 'L2_MAINNET',
}

const networkAddresses = {
  [ENetwork.L1_MAINNET]: {
    bondingManagerAddr: '0x511bc4556d823ae99630ae8de28b9b80df90ea2e',
  },
  [ENetwork.L2_MAINNET]: {
    bondingManagerAddr: '0x35Bcf3c30594191d53231E4FF333E8A770453e40',
  },
};

const getPendingData = async (
    orchAddr: string,
    network: ENetwork,
    claimBlock: number = 0,
) => {
  const roundNum = 2466; // does not matter on L2, only current round is used
  const provider = network === ENetwork.L1_MAINNET ? new EthersProviderWrapper(hre.companionNetworks['l1'].provider) : new EthersProviderWrapper(hre.network.provider)

.......
}
```

Reply:
refactored in 4467686